Rakefile

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+require 'rspec/core/rake_task'
+require 'puppet_fixtures/tasks'
```

Comment:
I'd like to avoid this:
```suggestion
```

Then in the tests you can use `rake --rakefile lib/puppet_fixtures/tasks.rb -T`